### PR TITLE
Update for No-proxy build-docker-image.sh

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 1]
```diff
@@ -23,7 +23,10 @@ With the trusted Big Data analytics and ML/DL support, users can run standard Sp
 
 ### 2.1 Prerequisite
 
-Download scripts and dockerfiles from [this link](https://github.com/intel-analytics/analytics-zoo/tree/master/ppml).
+Download scripts and dockerfiles from [this link](https://github.com/intel-analytics/analytics-zoo.git). And do the following commands:
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:26]: Remove .git

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md [hunk 2]
```diff
@@ -93,18 +105,29 @@ Enter `analytics-zoo/ppml/trusted-big-data-ml/scala/docker-graphene` dir.
 
 1. Copy `keys` and `password`
     ```bash
-    cd ppml/trusted-big-data-ml/scala/docker-graphene
+    cd trusted-big-data-ml/scala/docker-graphene
```
Comment (qiyuangong) [inline docs/readthedocs/source/doc/PPML/Overview/ppml.md:108]: Do we need to copy keys and password, if we use az graphene image?

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md (hunk 1 above)
Comment (ManfeiBai) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md]: Thanks, done~

File: docs/readthedocs/source/doc/PPML/Overview/ppml.md (hunk 2 above)
Comment (ManfeiBai) [reply docs/readthedocs/source/doc/PPML/Overview/ppml.md:108]: Thanks, no

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**